src/Security/Voter/Committee/EditDeleteCommitteeFeedItemVoter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace AppBundle\Security\Voter\Committee;
+
+use AppBundle\Committee\CommitteeFeedItemPermissions;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CommitteeFeedItem;
+use AppBundle\Security\Voter\AbstractAdherentVoter;
+
+class EditDeleteCommitteeFeedItemVoter extends AbstractAdherentVoter
```

Comment:
IMHO, I suggest you a name like `AdministrateCommitteeFeedItemVoter` or `ManageCommitteeFeedItemVoter`